tests/suite/resources_utils.py

```diff
@@ -227,6 +227,20 @@ def are_all_pods_in_ready_state(v1: CoreV1Api, namespace) -> bool:
     return pod_ready_amount == len(pods.items)
 
 
+def get_pods_amount(v1: CoreV1Api, namespace) -> int:
+    """
+    Get an amount of pods.
+
+    :param v1: CoreV1Api
+    :param namespace: namespace
+    :return: int
+    """
+    pods = v1.list_namespaced_pod(namespace)
+    if not pods.items:
+        return 0
+    return len(pods.items)
```

Comment:
This is not a real suggestion for you to implement, just a curiosity. Tis could be rewritten as:
```python
return 0 if not pod.items else len(pods.items)
```
In a more `pythonic` way